### PR TITLE
Fix #1048 - The accuracy module testing, fixes and readme

### LINKED ISSUE #1048: [Lake][Stats][App] Test app endpoint and document README.md for how to deploy/test it
### Background / motivation

As a result of deploying duckdb, the `accuracy/app` has been updated to query from the lake.

This means that at some point, we'll have to redeploy the app and we need to:

### TODOs / DoD
1. make sure the app endpoint is working as expected
2. the app needs to have a lake, that it's constantly updating the tables + etl
3. have instructions for how to deploy this
4. have instructions for how to test that it's working
5. make sure the predictoor.ai FE still works once the new server is deployed

Tasks:
- [ ] make sure the app endpoint is working as expected
- [ ] the app needs to have a lake, that it's constantly updating the tables + etl
- [ ] have instructions for how to deploy this
- [ ] have instructions for how to test that it's working
- [ ] make sure the predictoor.ai FE still works once the new server is deployed

### PR DESCRIPTION
**Pull request by kdetry**
Fixes #1048 

Changes proposed in this PR:

- [x] make sure the app endpoint is working as expected
- [x] the app needs to have a lake, that it's constantly updating the raw tables (not ETL)
- [x] have instructions for how to deploy this
- [x] have instructions for how to test that it's working
- [x] make sure the predictoor.ai FE still works once the new server is deployed

### CODE REVIEW

File: pdr_backend/accuracy/readme.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+## Flask API
+
+The `app.py` script provides a Flask endpoint with `/statistics` route to get the accuracy of the model.
+
+```bash
+curl http://localhost:5000/statistics
+```
+
+The endpoint will return the accuracy of the model in JSON format.
```
Comment (KatunaNorbert) [inline pdr_backend/accuracy/readme.md:40]: We could provide a JSON example of the response with all the fields and also mention that the API returns multiple values not just accuracy(`token_name, total_staked_yesterday, total_staked_today`), hence the statistics names.

Comment (kdetry) [reply pdr_backend/accuracy/readme.md:40]: Ok I will bring an example JSON and mention about it

Comment (kdetry) [reply pdr_backend/accuracy/readme.md:73]: I put the information to the "Usage" subtitle, under the `json` file info

File: pdr_backend/accuracy/readme.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+## Usage
+
+The `app.py` script is used to calculate the accuracy of a model based on the predicted and true values.
+
+```bash
+python pdr_backend/accuracy/app.py
+```
+
+
+The script uses the `GQLDataFactory` to fetch the predictions. Be sure to provide the correct values for `st_timestr` in the `ppss.yaml` file to get the correct predictions. It needs to be at least `28 days ago` from the current date.
```
Comment (KatunaNorbert) [inline pdr_backend/accuracy/readme.md:28]: Change 'to get correct predictions' to 'to get the required data for the calculations'

File: pdr_backend/accuracy/readme.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+```bash
+git clone <repository_url>
+cd pdr_backend
+
+pip install -r requirements.txt
+```
+
+## Usage
+
+The `app.py` script is used to calculate the accuracy of a model based on the predicted and true values.
```
Comment (KatunaNorbert) [inline pdr_backend/accuracy/readme.md:21]: 'to calculate the accuracy' -> 'to calculate the accuracy and other statistics'

**Comment by idiom-bytes** [comment]
**[Accuracy Update]**
@kdetry 
1. I really believe we should keep the `accuracy/` app as-is in main so we can get the duckdbpr ETL to where it needs to be
2. We can then focus on improving bronze-predictions and getting predictoor revenue complete

I would move all the latest `duckdb/app` code that uses lake/duckdb to another PR/branch and freeze that for later.

**Comment by idiom-bytes** [comment]
@kdetry as far as I can tell, this PR is to "fix" the Accuracy API w/ DuckDB and get the PR ready, but....
1. This PR does not include the original "Accuracy API"
2. This PR does not help me conclude the work with duckdb pr and get the PR ready

Please conclude your work here and get this PR ready to go before beginning work on 1001..

**Comment by idiom-bytes** [comment]
We're going to close this PR.

Rather than trying to merge the whole duckdb PR, we're going to merge it in phases.
https://github.com/oceanprotocol/pdr-backend/issues/1109

I'll be looking at this PR to potentially completing the integration.

Thank you!

**Pull request closed without merging**